### PR TITLE
Brand Consolidation - Add new veteran banner

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This pull request updates the veteran banner that appears along the homepage and hub landing pages.

### Tickets
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12006

## Testing done
Local testing using Chrome, responsive dev tool

### Desktop
image

### Mobile
image

## Acceptance criteria
- [ ] Footer meets visual standards on all viewports

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: va-gov/includes/veteran-banner.html
Comment (jbalboni) [reply va-gov/includes/veteran-banner.html]: Also, it looks like download both of these, even though we only use one, depending on the browser. Maybe we could get around that by using them as a background image?

**Comment by ncksllvn** [comment]
@jbalboni Updated to use background-image instead. From my network console, it looks like the background images are only downloaded as needed at each breakpoint.

**Comment by ncksllvn** [comment]
@RobAWilkinson I updated Formation here to 1.7.5 to get that last mixin. I won't merge until you've merged already with your work though, so if it causes conflicts I'll resolve them.

**Comment by ncksllvn** [comment]
PTAL @accbjt  @RobAWilkinson @jbalboni

**Review by jbalboni — APPROVED** [review]

File: src/platform/site-wide/sass/modules/_m-veteran-banner.scss
Comment (jbalboni) [inline src/platform/site-wide/sass/modules/_m-veteran-banner.scss]: Note that you could include this media query in the above class:

```scss
veteran-banner-image {
  margin: 0 auto;
  background-image: url(/img/homepage/veterans-banner-mobile.png);
  background-repeat: no-repeat;
  background-position: center bottom;
  background-size: 100% auto;
  height: $veteran-banner-mobile-height;
  max-width: $veteran-banner-mobile-width;
  @include media($small-screen) {
    background-image: url(/img/homepage/veterans-banner-desktop.png);
    height: $veteran-banner-desktop-height;
    max-width: $veteran-banner-desktop-width;
  }
}
```

I find that easier to read in general, since it keeps media queries nearby the styles they're modifying.

Comment (ncksllvn) [reply src/platform/site-wide/sass/modules/_m-veteran-banner.scss]: Ah nice, yeah after converting to background-image I didn't think of that.

**Comment by ncksllvn** [comment]
@amycesal Here's the updated homepage -

image

**Pull request merged**
